bin/eve.js

```diff
@@ -50,6 +51,10 @@ if(argv["help"]) {
     --port <number> Change the port the Eve server listens to (default 8080).
     --localControl  Entirely disable server interaction. File changes will be
                     stored in localStorage.
+    --json          Load the passed json file into @init. One additional
+                    colon separated argument to specify the target database,
+                    an additional to add the passed tag to each json record
+                    [[tag:]database]:filename
```

Comment:
Might be nice to invert the order here so the optionals come last in the order they'd get added. I don't feel super strongly on this. 3/10 maybe.